bin/www

```diff
@@ -84,7 +84,7 @@ function onError(error) {
 function onListening() {
   var addr = server.address();
   var bind = typeof addr === 'string'
-    ? 'pipe ' + addr
-    : 'port ' + addr.port;
-  debug('Listening on ' + bind);
+    ?  addr
+    : 'http://0.0.0.0:' + addr.port;
```

Comment:
Is there a way to dynamically determine the host? I don't know if the "box of donuts" works everywhere.

Reply:
AFAIK that means "listen on every interface" so it should be.... but maybe I could add an env var just like PORT to make it customizable?